src/vm/docs/mod.rs

```diff
@@ -430,10 +430,13 @@ const LET_API: SpecialAPI = SpecialAPI {
     output_type: "A",
     signature: "(let ((name1 expr1) (name2 expr2) ...) expr-body1 expr-body2 ... expr-body-last)",
     description: "The `let` function accepts a list of `variable name` and `expression` pairs,
-evaluating each expression and _binding_ it to the corresponding variable name. The _context_
-created by this set of bindings is used for evaluating its body expressions. The let expression returns the value of the last such body expression.
+evaluating each expression and _binding_ it to the corresponding variable name.
+`let` bindings are iterative: when a `let` binding is evaluated, it may refer to prior binding.
```

Comment:
The more accurate terminology here is that bindings are performed **sequentially** (rather than in parallel as in the original implementation).